### ISSUE TITLE
mym vs. python reshaping

### ISSUE THREAD

**Issue by fabiansinz** [datajoint/datajoint-python#191]
When storing arrays as blob in the database from matlab, mym stores it as column major, but datajoint-python loads it as row major. This means that inserting

```
     1     6    11    16
     2     7    12    17
     3     8    13    18
     4     9    14    19
     5    10    15    20
```

in Matlab will produce

```
array([[  1.,   2.,   3.,   4.],
       [  5.,   6.,   7.,   8.],
       [  9.,  10.,  11.,  12.],
       [ 13.,  14.,  15.,  16.],
       [ 17.,  18.,  19.,  20.]])
```

in Python. Solution is that blob needs to reshape arrays in column major mode.

**Comment by dimitri-yatsenko** [comment]
This was fixed in ce4cf77a6, right?

Ready to close?

**Comment by fabiansinz** [comment]
Yes.
